### DISCUSSION TITLE
Module build failed

### DISCUSSION THREAD

**Discussion by Bishwajitpvt** [Atri-Labs/atrilabs-engine#617] [Help: Installation & Start] [answered]
![image](https://user-images.githubusercontent.com/77453811/209202754-48ad1e1e-29d9-4ee4-96dc-2aa9bbe3b704.png)

**Comment by darshitac11** [comment, marked as answer]
Check solution [here](https://docs.atrilabs.com/troubleshoot/start_error#typeerror-cannot-read-properties-of-undefined-reading-text)
